_python_workout_code/7.1.py

```diff
@@ -0,0 +1,5 @@
+def  join_numbers(count):
+    list = [i for i in range(count)]
```

Comment:
`range()` already returns a sequence, so if you needed a list, you could have done: `list(range(count))`